packages/trip-details/src/fare-detail.tsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable import/prefer-default-export */
+import { Leg, Money } from "@opentripplanner/types";
+
+import React from "react";
+import styled from "styled-components";
+import { FormattedNumber } from "react-intl";
+import { FareDetailsProps } from "./types";
+
+interface FareTypeTableProps {
+  header: string;
+  cols: {
+    key: string;
+    header: string;
```

Comment:
What is the mechanism you have in mind to localize these headers? The keys are slightly different than what is defined in `TripDetails.fareKeyNameMap` prop, so the existing prop cannot be reused here.

Reply:
I think the best solution is to try to work on using `fareKeyNameMap` if possible -- @daniel-heppner-ibigroup do you have any thoughts on this or do you want to try to discuss offline

Reply:
`fareKeyNameMap` is not a localization object, unless I'm missing something? It's just a config object with the names, but there's no way of providing it on a per locale basis. As of now I don't believe we support localizing fare type names (beyond agency-specific names). For things like "ORCA" this is fine, but "Cash" and "Youth" should be localized.

I think the solution here is to add a new section to the localization files for fare names and then move all this (including the `fareKeyNameMap`) into there, so it can be localized.

Reply:
Could we create a map between `fareKeyNameMap` and language strings? So instead of putting the name in the map itself we instead use the key as a i18n key. If a result is returned, we override the name in the object. That would let us support localization here while also remaining backwards compatible

Reply:
So basically use the same key from `fareKeyNameMap` with i18n, and use the old `fareKeyNameMap` as a fallback? That sounds great to me. Want me to implement it?

Reply:
I can do it next week but if you have time before then that would be great!